src/main/java/roomescape/reservation/dto/ReservationWithPaymentResponse.java

```diff
@@ -0,0 +1,55 @@
+package roomescape.reservation.dto;
+
+import java.time.LocalDate;
+import roomescape.reservation.domain.ReservationStatus;
+import roomescape.reservation.domain.ReservationTime;
+import roomescape.reservation.domain.Theme;
+
+public class ReservationWithPaymentResponse {
+    private final Long id;
+    private final Theme theme;
+    private final LocalDate date;
+    private final ReservationTime time;
+    private final ReservationStatus status;
+    private final String paymentKey;
+    private final int totalAmount;
+
+    public ReservationWithPaymentResponse(Long id, Theme theme, LocalDate date, ReservationTime time,
+                                          ReservationStatus status, String paymentKey, int totalAmount) {
```

Comment:
```suggestion
    public ReservationWithPaymentResponse(
        Long id, 
        Theme theme, 
        LocalDate date, 
        ReservationTime time,
        ReservationStatus status, 
        String paymentKey, 
        int totalAmount
    ) {
```
지금 방식은 파라미터가 어떤게 있는지 파악하기 어려운것 같아요!
새로운 방식을 제안드립니다!

Reply:
이 부분 컨벤션 확인을 하지 못했네요 !
제안 해주신 방법으로 수정했습니다!